setup.py

```diff
@@ -70,7 +69,7 @@
     "learning": ['scikit-learn'],
     "gui-jupyter": ["hyperspy_gui_ipywidgets"],
     "gui-traitsui": ["hyperspy_gui_traitsui"],
-    "mrcz-blosc": ["blosc>=1.5"],
+    "mrcz": ["blosc>=1.5", 'mrcz>=0.3.6'],
```

Comment:
https://github.com/conda-forge/staged-recipes/pull/5670 just needs a rebuild and then it is ready to merge... but yes for this release this is not going to be ready!